code_folder/D1_model_components.py

```diff
@@ -140,6 +216,80 @@ def transformer_constant_efficiency_optimize(model, dict_asset, **kwargs):
         kwargs["transformers"].update({dict_asset["label"]: transformer})
         return
 
+    def transformer_constant_efficiency_multiple_buses_optimize(
+        model, dict_asset, **kwargs
+    ):
+        """
+
+        Parameters
+        ----------
+        dict_asset:
+        dictionary of the asset
+        kwargs:
+        other parameters, basically the busses components
+
+        Returns
+        -------
+
+        """
+        # check if the transformer has multiple input or multiple outpus busses
```

Comment:
is this a repition of gathering the multiple in/outbut busses in transformer_constant_efficiency_multiple_buses_fix? Then it could be shortened by defining an additional function.